### DISCUSSION TITLE
简悦 2.1 无法使用旧版方式的 PDF 导出功能（附解决方案）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1562] [重要通知]
#### 问题描述：

![image](https://user-images.githubusercontent.com/23144348/100861205-77a1d780-34cc-11eb-902e-b64d8731d956.png)
点这个不能导出pdf，提示我需要同步助手。
![image](https://user-images.githubusercontent.com/23144348/100861397-bafc4600-34cc-11eb-993c-87c7d70b14c9.png)
我已经关闭这个，问题依然存在

### 解决方案：

1. 使用离线版的用户，请重新下载 [离线版 2.1](http://ksria.com/simpread/crx/2.1.0/simpread.zip) **即可解决**。
2. **简悦 · 高级账户** 用户**请使用 2.1 版的原生导出方案**。
3. **简悦 · 基础账户** 不想使用 **方案1** 的话，需要使用 PDF 输出的话，请手动 **<kdb>Ctrl/Command + p</kbd>** 的方案。

**Comment by axtonliu** [comment]
看不懂! 请问如何"请使用 2.1 版的原生导出方案。"? 不想用同步助手!

**Reply by Kenshin** [reply]
> 请使用 2.1 版的原生导出方案。

意思就是说使用 **同步助手** 的方案。

如果你是免费用户的话，暂时请使用 `ctrl + p` 的浏览器自带方案。

**Reply by axtonliu** [reply]
谢谢这么快就回复. 我是Pro用户, 但不想用同步助手. 我主要就是为了导出PDF, 而同步助手在导出有些网页时, 生成的PDF只有一行英文错误提示. 因此想用回到以前的方式.

**Reply by Kenshin** [reply]
> 生成的PDF只有一行英文错误提示

建议给个 URL 看看，另外这个问题在特定的页面才出现吗？

> 但不想用同步助手

目前 2.1 版就使用 `ctrl + p` 的方案。（ 1.x 版就用了这个方式，只是程序自动调用了 `ctrl + p` 而已，2.1 有个 Bug 导致程序无法自动调用它，所以需要手动。

新版（在路上）已经修复了这个 Bug

另外，`ctrl + p` 的方式有些图会被截断，所以最好的方案就是同步助手。

**Reply by axtonliu** [reply]
https://www.scientificamerican.com/article/did-we-receive-a-message-from-a-planet-orbiting-the-nearest-star/
错误信息是: Error: ENOENT: no such file or directory ....
只在特定的页面才出现, 我昨天看了6~7个网页, 其中2个出现了问题.

另外, 有些网页为了防止Copy内容, 屏蔽了鼠标选取文字的操作. 使用简悦阅读模式后, 选取文字还是被屏蔽的, 不知道能不能在阅读模式后, 把屏蔽鼠标选取文字的代码给它清理掉? 否则没有办法做高亮标注. 另外, 类似这种页面进入稍后读后, 再进入阅读模式, 会显示解析正文失败的错误.  

这些是前一阵发现的问题, 没有存URL. 我也没有时间进行更多测试. 无法给你提供进一步信息了.

**Reply by axtonliu** [reply]
> 另外，`ctrl + p` 的方式有些图会被截断，所以最好的方案就是同步助手。

这点确是是个大问题. 所以大多数情况我都是剪藏. 而同步助手, 说实话我觉得太繁琐了. 包括简悦本身我都觉得有些繁琐, 其实建议参考参考Diigo类的软件的优缺点, 把标注和标注管理做精就更好了, 不需要太多功能. 

只是个人的一些浅见, 仅供参考.

**Reply by Kenshin** [reply]
@axtonliu 

需求不太一样，Diigo 类的产品问题是数据不在用户自己「手里」，想想用 Diigo 在导入到其它服务里面，有多不方便就知道了。
操作繁琐，像 marginnote DEVOThink 这类的产品至少还有上手难度呢，所以从另外一个角度看简悦就知道它并不繁琐了。

不要单纯的把它当作 剪藏 或 阅读模式 的产品就好了，它是一个知识管理范畴的产品。

当然，简化流程这个事情是一定做下去的。

**Comment by axtonliu** [comment]
离线版 2.1 是个2.0版本的链接!

**Reply by Kenshin** [reply]
谢谢提示，已 Fix 链接问题。
